cmd/opentelemetry/app/exporter/elasticsearchexporter/esmodeltranslator/modeltranslator.go

```diff
@@ -75,18 +75,18 @@ func (c *Translator) ConvertSpans(traces pdata.Traces) ([]ConvertedData, error)
 	if rss.Len() == 0 {
 		return nil, nil
 	}
-	containers := make([]ConvertedData, 0, traces.SpanCount())
+	spansData := make([]ConvertedData, 0, traces.SpanCount())
```

Comment:
Just checking; did you intend to rename this to `spanData` as per the PR title?

Reply:
yes